gno.land/cmd/gnoweb/main_test.go

```diff
@@ -1,14 +0,0 @@
-package main
```

Comment:
Please keep it this way: 1. to facilitate the addition of second tests later, and 2. to ensure that `go test` does not skip this folder and passes the basic syntax checks.

Reply:
I cannot _exactly_ keep it this way as I'm using the `command` package. But I found another way to implement it (I personally think it's even better as I test the whole setup of the main).